databases/week3/databaseWeek3.sql

```diff
@@ -0,0 +1,244 @@
+    '2023-03-22', 5, 25.00, NOW()
+  );
+VALUES 
+  (
+    3, 'Argentinian Lunch', 'Traditional asado (BBQ), empanadas and fernet', 
+    'Mitre, S2000 Rosario, Santa Fe, Argentina', 
+    '2023-03-25', 20, 100.00, NOW()
+  );
+-- Queries for meal table
+-- Get all meals
+SELECT 
+  * 
+FROM 
+  Meal;
+-- Add a new meal
+INSERT INTO Meal (
+  `id`, `title`, `description`, `location`, 
+  `when`, `max_reservations`, `price`, 
+  `created_date`
+) 
+VALUES 
+  (
+    4, 'American classic', 'Mc and cheese', 
+    'Tryon, North Carolina 28016, USA', 
+    '2023-03-30', 3, 100.00, NOW()
+  );
+-- Get a meal with any id, fx 1
+SELECT 
+  * 
+FROM 
+  Meal 
+WHERE 
+  id = 1;
+-- Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE 
+  Meal 
+SET 
+  title = 'Fantastic Italian Dinner' 
+WHERE 
+  id = 1;
+-- Delete a meal with any id, fx 1:
+DELET 
+FROM 
+  Meal 
+WHERE 
+  id = 1;
+-- Queries for Reservation
+-- Get all reservations
+SELECT 
+  * 
+FROM 
+  Reservation;
+-- Add a new reservation
+INSERT INTO Reservation (
+  `id`, `title`, `number_of_guests`, 
+  `meal_id`, `created _date`, `contact_phonenumber`, 
+  `contact_name`, `contact_email`
+) 
+VALUES 
+  (
+    1, 'Birthday celebration', 15, 1, NOW(), 
+    '55228591', 'Mercedes Ubeira', 'mercedes@example.com'
+  );
+-- Get a reservation with any id, fx 1:
+SELECT 
+  * 
+FROM 
+  Reservation 
+WHERE 
+  id = 1;
+-- Update a reservation with any id, fx 1. Update any attribute fx the title or multiple attributes:
+UPDATE 
+  Reservation 
+SET 
+  title = 'Surprise birthday celebration', 
+  number_of_guests = 3 
+WHERE 
+  id = 1;
+-- Delete a reservation with any id, fx 1:
+DELETE FROM 
+  Reservation 
+WHERE 
+  id = 1;
+-- Queries for review table
+-- Get all Reviews
+SELECT 
+  * 
+FROM 
+  Review;
+-- Add a new review
+INSERT INTO Review (
+  `title`, `description`, `meal_id`, 
+  `stars`
+) 
+VALUES 
+  (
+    'Delicious', 'Really good food and the hosts were super kind', 
+    1, 5
+  );
+-- Get a review with any id, fx 1
+SELECT 
+  * 
+FROM 
+  Review 
+WHERE 
+  id = 1;
+-- Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes:
+UPDATE 
+  Review 
+SET 
+  title = 'So much hygge!', 
+  description = 'Incredible food with good hosts.' 
+WHERE 
+  id = 1;
+-- Delete a review with any id, fx 1:
+DELETE FROM 
+  Review 
+WHERE 
+  id = 1;
+-- Get meals that have a price smaller than a specific price (e.g. 90):
+SELECT 
+  * 
+FROM 
+  Meal 
+WHERE 
+  price < 90;
+-- Get meals that still have available reservations:
+SELECT 
+  * 
+FROM 
+  Meal 
+WHERE 
+  max_reservations > (
+    SELECT 
+      COUNT(*) 
+    FROM 
+      Reservation 
+    WHERE 
+      Reservation.meal_id = Meal.id
+  );
+-- Get meals that partially match a title. (e.g. "Rød grød med" will match the meal with the title "Rød grød med fløde"):
+SELECT 
+  * 
+FROM 
+  Meal 
+WHERE 
+  title LIKE '%Rød grød med';
```

Comment:
This title won't match "Rød grød med fløde" because you've added % only at the beginning which means your query will give you all strings that and with "Rød grød med".